client/src/hooks/queries/karmaGap/useMilestonesPerGrantPerProgram.ts

```diff
@@ -17,9 +17,14 @@ export default function useMilestonesPerGrantPerProgram(
     select: response =>
       response.data.find(
         element =>
-          element.project.externalAddresses?.octant.toLowerCase() === projectAddressToLowerCase ||
-          element.project.recipient.toLowerCase() === projectAddressToLowerCase ||
-          element.project.details.recipient.toLowerCase() === projectAddressToLowerCase,
+					element.project?.externalAddresses?.octant?.toLowerCase() ===
+						projectAddressToLowerCase ||
+					element?.externalAddresses?.octant?.toLowerCase() ===
+						projectAddressToLowerCase ||
```

Comment:
`externalAddresses.octant` is/can be in two places at the same time?

Reply:
Yes, they can.

Reply:
@amaurymagalhaes: it can? What is the scenario when it's in `element.externalAddresses` but not in `element.project.externalAddresses`?